core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -150,6 +150,7 @@
   MASTER_WEB_HOSTNAME(Name.MASTER_WEB_HOSTNAME, null),
   MASTER_WEB_PORT(Name.MASTER_WEB_PORT, 19999),
   MASTER_WHITELIST(Name.MASTER_WHITELIST, "/"),
+  MASTER_CONNECTION_TIMEOUT_MS(Name.MASTER_CONNECTION_TIMEOUT_MS, 0),
```

Comment:
To be compatible with the original version of the alluxio, the default is set to 0.To solve this problem you need to set this parameter to greater than 6000